### ISSUE TITLE
Add latest macOS IPSW link to feed

### ISSUE THREAD

**Issue by kevinmcox** [macadmins/sofa#67]
Great project!
The link to the `InstallAssistant.pkg` is helpful.
~~Would it be possible to include the link to the `.IPSW` file as well for each release?~~

Would it be possible to include the link to the `.IPSW` file for the latest (most recent) macOS release?

**Comment by arubdesu** [comment]
Hey @kevinmcox, we had teased that in the README but I honestly wouldn't have known where to exactly start, since we want to use canonical resources and not sites like ipsw.me. Luckily I asked @ninxsoft and he gave me a pointer about how I can at least maybe pull the latest? I'll follow up as I can investigate more.

**Comment by kevinmcox** [comment]
Apologies, I'm definitely guilty of not reading close enough and missing that callout. I'm glad it is already on the radar!

**Comment by arubdesu** [comment]
@kevinmcox I've discussed this idea with @headmin and so I thought I'd try it out on this request: can you be a 'verifier' and confirm this feature is delivered as hoped/desired once the PR lands? **Could you also therefore edit this issue** to make it explicit if you are ok with just latest macOS for now? That way we can historically track what happened when and who 'vouched' for it getting delivered, if that makes sense?
I'll follow up with more context in Slack if that's OK

**Comment by kevinmcox** [comment]
Hey @arubdesu, absolutely. I can always open additional issues as needed. (For instance, if beta releases eventually get added it would be great to see IPSW included with them as well.) But I'll clean this one up to be very specific.

**Comment by arubdesu** [comment]
Pardon I went ahead and edited it, just FYI it will be a separate task to display it in the GUI (which I'll leave to @headmin) and this issue should at least ping you when it's live in the feed

**Comment by arubdesu** [comment]
As usual I'm leaving integrating the download link into the GUI to @headmin, but we're live with latest!
